commcare_connect/opportunity/views.py

```diff
@@ -1120,7 +1133,10 @@ def user_visit_review(request, org_slug, opp_id):
     user_visit_reviews = UserVisit.objects.filter(opportunity=opportunity, review_created_on__isnull=False).order_by(
         "visit_date"
     )
-    table = UserVisitReviewTable(user_visit_reviews, org_slug=request.org.slug)
+    review_filter = UserVisitReviewFilter(request.GET, queryset=user_visit_reviews)
+    table = UserVisitReviewTable(
+        review_filter.qs, org_slug=request.org.slug, template_name="django_tables2/bootstrap5.html"
```

Comment:
Same template question

Reply:
Added the template to the table class. [da4f59e](https://github.com/dimagi/commcare-connect/pull/475/commits/da4f59e3a2c6f5803dd4e5682ef0cbea3aa1d3d0)